Buildings/Fluid/DXSystems/Heating/AirSource/Data/Generic/Coil.mo

```diff
@@ -0,0 +1,151 @@
+within Buildings.Fluid.DXSystems.Heating.AirSource.Data.Generic;
+record Coil
+  "Performance record for a DX Heating Coil with one or multiple stages"
+  extends Modelica.Icons.Record;
+
+  final parameter Boolean sinStaOpe = nSta == 1
+    "The data record is used for single speed operation"
+    annotation(HideResult=true);
+
+  parameter Integer nSta(
+    final min=1)
+    "Number of stages"
+    annotation (Dialog(enable = not sinStaOpe));
```

Comment:
For heating coil, we only allow one stage, so this record should not allow multiple stages.